src/main/java/yevhenii/lostfilmdemo/services/impl/ConvertServiceImpl.java

```diff
@@ -0,0 +1,44 @@
+package yevhenii.lostfilmdemo.services.impl;
+
+import org.springframework.stereotype.Service;
+import yevhenii.lostfilmdemo.entity.TVSeries;
+import yevhenii.lostfilmdemo.rss.FeedMessage;
+import yevhenii.lostfilmdemo.services.ConvertService;
+
+@Service
+public class ConvertServiceImpl implements ConvertService {
+
+    private String getName(StringBuilder str){
+        return str.substring(str.indexOf("(")+1, str.indexOf(")"));
+    }
+    private String getRussianName(StringBuilder str){
```

Comment:
instead of all this methods which are parsing String use regex expression with group for each value you wanna to extract